packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -264,6 +264,7 @@ function formatPostInChannel(post: Post, previousPost: Post | undefined | null,
 
 export function makeGetPostsInChannel(): (state: GlobalState, channelId: $ID<Channel>, numPosts: number) => PostWithFormatData[] | undefined | null {
     return createSelector(
+        'makeGetPostsInChannel',
```

Comment:
Unfortunately I couldn't find an alternative to passing in the function names here. I explored `Function.caller` and other related options but none of the reasonable ones I found worked in strict mode.

Reply:
Yeah, I fought with that for a while, but I couldn't find any way to do it automatically either. With all of the weird pseudo-metaprogramming things you can do in JavaScript, I'm surprised that doesn't exist at all.